apps/site/components/Downloads/DownloadReleasesTable/index.tsx

```diff
@@ -21,35 +20,33 @@ const DownloadReleasesTable: FC = async () => {
           <th>{t('components.downloadReleasesTable.version')}</th>
           <th>{t('components.downloadReleasesTable.nApiVersion')}</th>
           <th>{t('components.downloadReleasesTable.codename')}</th>
-          <th>{t('components.downloadReleasesTable.releaseDate')}</th>
-          <th>{t('components.downloadReleasesTable.npmVersion')}</th>
+          <th>{t('components.downloadReleasesTable.firstReleased')}</th>
+          <th>{t('components.downloadReleasesTable.lastUpdated')}</th>
+          <th>{t('components.downloadReleasesTable.status')}</th>
           <th></th>
         </tr>
       </thead>
       <tbody>
         {releaseData.map(release => (
           <tr key={release.major}>
-            <td data-label="Version">v{release.version}</td>
+            <td data-label="Version">v{release.major}</td>
             <td data-label="Modules">v{release.modules}</td>
             <td data-label="LTS">{release.codename || '-'}</td>
+            <td data-label="Date">
+              <time>{release.currentStart}</time>
+            </td>
             <td data-label="Date">
               <time>{release.releaseDate}</time>
             </td>
-            <td data-label="npm">v{release.npm}</td>
-            <td className="download-table-last">
-              <LinkWithArrow
-                href={`https://nodejs.org/download/release/${release.versionWithPrefix}/`}
+            <td data-label="Status">
+              <Badge
+                kind={release.status === 'End-of-life' ? 'warning' : 'default'}
               >
-                {t('components.downloadReleasesTable.actions.releases')}
-              </LinkWithArrow>
-
-              <LinkWithArrow href={`${BASE_CHANGELOG_URL}${release.version}`}>
-                {t('components.downloadReleasesTable.actions.changelog')}
-              </LinkWithArrow>
-
-              <LinkWithArrow href={getNodeApiLink(release.versionWithPrefix)}>
-                {t('components.downloadReleasesTable.actions.docs')}
-              </LinkWithArrow>
+                {release.status}
```

Comment:
Is there any way to indicate active vs maintenance LTS?  

If not, this isn't a major issue